hey.go

```diff
@@ -135,6 +135,10 @@ func main() {
 	// set content-type
 	header := make(http.Header)
 	header.Set("Content-Type", *contentType)
+
+	// set a default user-agent
+	header.Set("User-Agent", "hey (https://github.com/rakyll/hey)")
```

Comment:
Need is a strong word here, but as a sysadmin I think a User-Agent is a good thing to have. When I see a lot of requests coming up in a logfile its nice to know what's making them; instantly I "know" if they are legit or not.

Right now, the user-agent sent by Hey is the default golang user-agent (Go-http-client/1.1) which doesn't say much about what's making the request and if the user of Hey wants to use another user-agent (if the application being tested requires it for instance) he/she can do that by using -H "User-Agent: whatever".